app/factories/bulkrax/valkyrie_object_factory.rb

```diff
@@ -266,6 +266,7 @@ def create_work(attrs)
 
     def combined_files_with(remote_files:)
       thumbnail_url = self.attributes['thumbnail_url']
+      return [thumbnail_url] if remote_files.blank?
```

Comment:
to clarify, if thumbnail_url is also blank it'll return an empty array?

Reply:
An empty array should avoid the error (the error was trying to merge the thumbnail url into a nil remote_files value.) 

But would be a good thing to double-check.